langchain/chains/qa_with_sources/__init__.py

```diff
@@ -49,9 +49,13 @@ def _load_map_reduce_chain(
     collapse_llm: Optional[BaseLLM] = None,
     **kwargs: Any,
 ) -> MapReduceDocumentsChain:
-    map_chain = LLMChain(llm=llm, prompt=question_prompt)
+    map_chain = LLMChain(
+        llm=llm, prompt=question_prompt, verbose=kwargs.get("verbose", False)
+    )
     _reduce_llm = reduce_llm or llm
-    reduce_chain = LLMChain(llm=_reduce_llm, prompt=combine_prompt)
+    reduce_chain = LLMChain(
+        llm=_reduce_llm, prompt=combine_prompt, verbose=kwargs.get("verbose", False)
+    )
     combine_document_chain = StuffDocumentsChain(
```

Comment:
probably want to pass it in here as well

Reply:
`StuffDocumentsChain` takes in `reduce_chain` llm which has verbose arguments set.